cdptools/bin/process_single_event.py

```diff
@@ -0,0 +1,65 @@
+import argparse
+import sys
+import logging
+import traceback
+from pathlib import Path
+
+from cdptools import get_module_version
+from ..dev_utils import load_custom_object
+from ..event_scrapers import seattle_event_scraper as scraper
+
+logging.basicConfig(
+    level=logging.INFO,
+    format="[%(levelname)4s: %(module)s:%(lineno)4s %(asctime)s] %(message)s"
+)
+log = logging.getLogger(__name__)
+
+###############################################################################
+
+
+class Args(argparse.Namespace):
+
+    def __init__(self):
+        self.__parse()
+
+    def __parse(self):
+        p = argparse.ArgumentParser(
+            prog="process_single_event",
+            description="Process a single event."
+        )
+
+        p.add_argument("--event_url", type=str, help="URL of the event you want to process.", required=True)
+        p.add_argument("--config_path", type=Path, help="Path to a configuration file with details for the pipeline.",
+                       required=True)
+        p.add_argument("--debug", action="store_true", dest="debug", help="Show traceback if the script were to fail.")
+        p.parse_args(namespace=self)
+
+
+def main():
+    args = Args()
+
+    try:
+        event_scraper = scraper.SeattleEventScraper()
```

Comment:
This module should be loaded with the config file. `load_custom_object`

Reply:
Done! Didn't see that EventGatherPipeline sets a scraper as an instance variable